app/models/benefit_model.rb

```diff
@@ -5,6 +5,51 @@ class BenefitModel
 
   FREQUENCY_OPTIONS = %w[every_week every_two_weeks every_four_weeks monthly].freeze
 
+  PASSPORTED_BENEFITS = [
+    "Universal Credit",
+    "Income Support",
+    "Income-based Jobseeker's Allowance (JSA)",
+    "Income-related Employment and Support Allowance (ESA)",
+    "Guarantee Credit element of Pension Credit",
+  ].freeze
+
+  DISREGARDED_BENEFITS = [
```

Comment:
This list doesn't quite match up to what CFE returns. E.g. CFE has `Carer's Allowance` which is one character different from `Carer’s Allowance` which is here.

The good news is that CFE also returns an `exclude_from_gross_income` attribute for each benefit, and if we ignore every item that has that set to true, we exclude all the below from the list.